crawl-ref/source/tileweb.cc

```diff
@@ -524,6 +524,11 @@ wint_t TilesFramework::_handle_control_message(sockaddr_un addr, string data)
         // XX only set this flag if a relevant option has actually changed
         Options.prefs_dirty = true;
     }
+    else if (msgtype == "main_menu_action"
```

Comment:
eventually it would be better to have a control message type that can send arbitrary commands (possibly just as a string, like the lua API for this). This doesn't need to happen in this PR, but it'd be worth adding a TODO note to this effect near this message type

Reply:
Done. If there is anything else I should change, please let me know.